frontend/src/components/communication/CommentInput.js

```diff
@@ -52,6 +54,7 @@ function CommentInput({
   hasComments,
   infoTextSize,
   useIconButton,
+  defaultProjectCommentInput,
```

Comment:
It seems like nothing is ever being passed through this prop. E.g. in `IdeaCommentsSection` it is passed as className (`className={classes.defaultProjectCommentInput}`) even though `CommentInput` doesn't take a className so this just does nothing. If it's not required I'd just remove it everywhere. If it is required I'd check if it really needs to be generic

Reply:
Removed